docs/install.rst

```diff
@@ -113,32 +112,19 @@ To find the JDK, Rally expects the environment variable ``JAVA_HOME`` to be set
 Installing Rally
 ----------------
 
-Simply install Rally with pip: ``pip3 install esrally``
-
-.. note::
-
-   Depending on your system setup you may need to prepend this command with ``sudo``.
+1. Ensure ``~/.local/bin`` is in your ``$PATH``.
+2. Install Rally: ``python3 -m pip install --user esrally``.
```

Comment:
Alternatively `--user` can be skipped and then `esrally` lives (in the form of a pyenv shim) in `$ which esrally
~/.pyenv/shims/esrally`. In this case the user would still need to either logout/login or `exec $SHELL` to make the shim active.

Since we are anyway upgrading pip (earlier) in a `--user` context, I think we should stick to that convention, which IMHO is a better practice, cleaner approach and ultimately a more discoverable location (`~/.local/bin`) for binaries to be stored. The requirement to have `~/.local/bin` is also IMHO a best practice that users should follow.

All these instructions have been tested -- at least -- in CentOS 8 and Ubuntu 18.04. More testing very welcome.

Reply:
for some reason `python3 -m pip install --user esrally` keeps installing 1.4.1 rally for me

Reply:
oh i think that's because python version is 3.6 not 3.8